source/backend/api/Services/PropertyService.cs

```diff
@@ -349,36 +332,41 @@ public PimsProperty PopulateNewProperty(PimsProperty property, bool isOwned = fa
             var boundaryGeom = property.Boundary;
             if (boundaryGeom != null && boundaryGeom.SRID != SpatialReference.BCALBERS)
             {
-                var newCoords = property.Boundary.Coordinates.Select(coord => _coordinateService.TransformCoordinates(boundaryGeom.SRID, SpatialReference.BCALBERS, coord));
-                var gf = NetTopologySuite.NtsGeometryServices.Instance.CreateGeometryFactory(SpatialReference.BCALBERS);
-                property.Boundary = gf.CreatePolygon(newCoords.ToArray());
+                _coordinateService.TransformGeometry(boundaryGeom.SRID, SpatialReference.BCALBERS, boundaryGeom);
```

Comment:
Does it make sense to do this here, or should we just call the new UpdateLocation method?

Reply:
The **UpdateLocation** calls the property repository to- well - update the property in the DB. This method is called when we are adding a brand new one as part of a larger parent entity (say an acquisition file that gets a brand new property added). 

Do you have a suggestion on how to reuse the UpdateLocation to not call the DB?